src/Pages/Organisation/Organisations.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import OrganisationsList from '../../Components/OrganisationsList/OrganisationsList';
+import { Helmet } from "react-helmet";
+import './Organisations.scss'
+import il_orgs from '../../Assets/il_orgs.svg';
+import FilterSidebar from '../../Components/FilterSidebar/FilterSidebar';
+import SearchBar from '../../Components/SearchBar/SearchBar';
+
+// Initial filters state
+const initialFilters = {
+  categories: [],
+  technology: [],
+  year: [],
+  program: [],
+};
+
+// Data for filters
+const data = {
+  categories: ['Open Source', 'Web Development', 'Mobile Development', 'Healthcare', 'Spatial Computing', 'Mathematics', 'Package Management', 'Environmental Science', 'Women in Computing'],
+  technology: ['React', 'Angular', 'Vue', 'Python', 'C++', 'Java', 'JavaScript', 'Node.js', 'Ruby', 'Shell'],
+  year: ['2020', '2021', '2022'],
+  program: ['GSOC', 'Outreachy', 'Season of Docs'],
+};
+
+
+// Organisations data
+const organisationsData = [
+  {
+    id: 1,
+    name: 'Inkscape',
+    categories: 'End-user applications',
+    technology: ['React', 'C++', 'Python', 'C'],
+    desc: 'Inkscape is a professional vector graphics editor for Windows, Mac OS X and Linux. It\'s free and open source.',
+    year: ['2022']
+  },
+  {
+    id: 2,
+    name: 'Postman',
+    categories: 'API Development',
+    technology: ['API', 'Javascript', 'Node.js'],
+    year: ['2021']
+  },
+  {
+    id: 3,
+    name: 'OpenMRS',
+    categories: 'Healthcare',
+    technology: ['Java', 'React', 'MySQL'],
+    year: ['2020']
+  },
+  {
+    id: 4,
+    name: 'Liquid Galaxy',
+    categories: 'Spatial Computing',
+    technology: ['Python', 'C++', 'HTML/CSS'],
+    year: ['2022']
+  },
+  {
+    id: 5,
+    name: 'SymPy',
+    categories: 'Mathematics',
+    technology: ['Python', 'C'],
+    year: ['2021']
+  },
+  {
+    id: 6,
+    name: 'FOSSASIA',
+    categories: 'Open Tech Development',
+    technology: ['Python', 'JavaScript', 'Angular'],
+    year: ['2020']
+  },
+  {
+    id: 7,
+    name: 'Homebrew',
+    categories: 'Package Management',
+    technology: ['Ruby', 'Shell'],
+    year: ['2022']
+  },
+  {
+    id: 8,
+    name: 'Public Lab',
+    categories: 'Environmental Science',
+    technology: ['JavaScript', 'React', 'Python'],
+    year: ['2021']
+  },
+  {
+    id: 9,
+    name: 'Systers',
+    categories: 'Women in Computing',
+    technology: ['Python', 'Java', 'C++'],
+    year: ['2020']
+  },
+];
```

Comment:
I hope all of this is just for testing. Adding a comment so that we an remove it before merging. Should come from Firebase.